siliconcompiler/core.py

```diff
@@ -675,6 +675,16 @@ def load_checklist(self, name):
         else:
             self.error(f'Checklist module {name} not found in $SCPATH or siliconcompiler/checklists/.')
 
+    ##########################################################################
+    def use(self, module):
+        '''
+        Loads a SiliconCompiler module into the current chip object by calling
+        a module.setup() method.
+        '''
+
+        self.import_library(module.setup())
```

Comment:
Maybe add a type check (and check if `.setup()` is present) to provide a user friendly error if they call this incorrectly?